webapp/channels/src/sass/components/_buttons.scss

```diff
@@ -298,6 +302,20 @@ button {
             }
         }
     }
+    &.btn-tertiary.btn-inverted {
+        background-color: rgba(var(--sidebar-text-rgb), 0.12);
+        color: rgba(var(--sidebar-text-rgb), 1);
+
+        &:hover {
+            background-color: rgb(var(--sidebar-text-rgb), 0.16);
+        }
+
+        &:active,
+        &[aria-expanded="true"][aria-haspopup="true"] {
```

Comment:
Ooh, this line is really nice.

A few days ago, I was fighting with the janky logic that keeps the post highlighted when a menu inside of it is ago, and I was thinking of using `:focus-within` or something similar, but I hadn't thought of looking at the ARIA attributes to determine if the menu is open. Now, I'm thinking we might be able to use something like `.Post:has([aria-expanded="true"][aria-haspopup="true"])` so that the `PostComponent` won't have to know whether or not there's a menu inside of it

Reply:
Do you want to open ticket for the postcomponent?

Reply:
I filed one: MM-62675